cypress/e2e/calendar/sort-headings-all-calendars.cy.js

```diff
@@ -0,0 +1,178 @@
+import {
+  MultiColumnListHeader, Pane, Link
+} from '../../../interactors';
+import calendarFixtures from '../../fixtures/calendar_e2e_fixtures';
+import CalendarSortTableTools from '../../support/utils/uiCalendar_SortTableTools';
+import { assertCalendarsIsProperlySorted } from '../../support/fragments/calendar/sort-table';
+
+const testServicePoint = calendarFixtures.servicePoint;
+const testCalendar = calendarFixtures.calendar;
+
+const addSort = CalendarSortTableTools.sortableMultiColumnList.addSort;
+const sortCalendars = CalendarSortTableTools.sortAllCalendars.sortCalendars;
+
+
+
+describe('Sort headings on "All calendars" tab', () => {
+  let testCalendarResponse;
+  before(() => {
+    // login as admin so necessary state can be created
+    cy.loginAsAdmin();
+
+    // get admin token to use in okapiRequest to retrieve service points
+    cy.getAdminToken();
+
+    // reset db state
+    cy.deleteServicePoint(testServicePoint.id, false);
+
+    // create test service point
+    cy.createServicePoint(testServicePoint, (response) => {
+      testCalendar.assignments = [response.body.id];
+
+      cy.createCalendar(testCalendar, (calResponse) => {
+        testCalendarResponse = calResponse.body;
+      });
+      cy.openCalendarSettings();
+    });
+  });
+
+  after(() => {
+    cy.deleteServicePoint(testServicePoint.id, true);
+    cy.deleteCalendar(testCalendarResponse.id);
+  });
+
+
+  it('sorts headings on "All calendars" tab', () => {
```

Comment:
it('{test case id from test rail} {test case name from test rail} (vega)', { tags: [TestTypes.smoke, devTeams.vega] }, () => {